compiler/util/files.cpp

```diff
@@ -740,10 +741,11 @@ void codegen_makefile(fileinfo* mainfile, const char** tmpbinname, bool skip_com
   genCFiles(makefile.fptr);
   genObjFiles(makefile.fptr);
   fprintf(makefile.fptr, "\nLIBS =");
-  forv_Vec(const char*, dirName, libDirs)
+  for_vector(const char, dirName, libDirs)
     fprintf(makefile.fptr, " -L%s", dirName);
-  forv_Vec(const char*, libName, libFiles)
+  for_vector(const char, libName, libFiles) {
     fprintf(makefile.fptr, " -l%s", libName);
+  }
```

Comment:
Just curious, why the curlies here but not in the previous one?

Reply:
Uh..... I think I must've added some debug printing at some point and then removed it without noticing I hadn't "cleaned it back up" again.